app/code/Magento/Catalog/Model/Product/Type.php

```diff
@@ -24,6 +24,9 @@ class Type implements OptionSourceInterface
     const TYPE_BUNDLE = 'bundle';
```

Comment:
these type constants do already exist and should be reused from there
for example: \Magento\Bundle\Model\Product\Type::TYPE_CODE